client/thin-replica-client/include/client/thin-replica-client/trs_connection.hpp

```diff
@@ -31,21 +31,25 @@ struct TrsConnectionConfig {
   // use_tls determines if a TLS enabled secure channel will be opened
   // by the thin replica client, or an insecure channel will be employed.
   const bool use_tls;
-  // thin_replica_tls_cert_path specifies the path of the certificates used for
-  // the TLS channel.
-  const std::string thin_replica_tls_cert_path;
   // client_key is the private key used by TRC to create a secure channel if
   // `is_insecure_trc_val` is false.
-  const std::string client_key;
-  const std::string client_cert_path;
-  const std::string server_cert_path;
-
-  TrsConnectionConfig(bool use_tls_, const std::string& thin_replica_tls_cert_path_, const std::string& client_key_)
+  const std::string& client_key;
+  const std::string& client_cert;
+  const std::string& server_cert;
+  // If TLS is enabled for TRC-TRS connection, the client cert must have the
+  // client ID in the OU field which should match the client ID of the client
+  const std::string& cert_client_id;
```

Comment:
Similar to the comment above. If the user provides this field then we don't know where it actually comes from. Instead call it `id` or `client_id`.